cppfd/parallel_mesh.cpp

```diff
@@ -259,6 +261,7 @@ LocalToGlobalPointIndices(const LocalCoordinates& loc) const{
   return {m, n};
 }
 
+#ifdef OUTPUT_VTK_FILES
```

Comment:
Same problem here

Reply:
@ppebay working for both cases for me, please check. Thanks!